### PR TITLE
Added default rates history

### PR DESCRIPTION
**Pull request by Raibo**

### CODE REVIEW

**Comment by Raibo** [comment]
Currently no migration added, some problems with it.

File: HappyTravel.CurrencyConverterApi/Services/RateService.cs [hunk 1]
```diff
@@ -138,6 +138,15 @@ private static TimeSpan GetTimeSpanToNextHour()
         }
 
 
+        private static TimeSpan GetTimeSpanToNextMinute()
```
Comment (kirillta) [inline HappyTravel.CurrencyConverterApi/Services/RateService.cs:141]: This method may be moved to an extension class

**Review by kirillta — APPROVED** [review]

**Review by mingazhev — APPROVED** [review]
Need make migration work

File: HappyTravel.CurrencyConverterApi/Services/RateService.cs (hunk 1 above)
Comment (Raibo) [reply HappyTravel.CurrencyConverterApi/Services/RateService.cs:141]: What instance should I extend with this new extension method?

**Pull request merged**